components/src/atoms/StyledText/__tests__/StyledText.test.tsx

```diff
@@ -226,7 +226,7 @@ describe('StyledText', () => {
       children: 'h3Bold',
     }
     render(props)
-    expect(screen.getByText('h3Bold')).toHaveStyle(
+    expect(screen.queryAllByText('h3Bold')[0]).toHaveStyle(
```

Comment:
We generally prefer when rendering only one element for the sake of clarity.
```suggestion
    expect(screen.getByText('h3Bold')).toHaveStyle(
```